src/utils/createDate.ts

```diff
@@ -0,0 +1,10 @@
+const milliseconds = 1000;
+
+/**
+ * JavaScript uses milliseconds since EPOCH, therefore the value
+ * used to create the new Date instance needs to be multiplied by 1000
+ * to create an accurate dateTime
+ */
+export function createDate(seconds: number): Date {
+  return new Date(seconds * milliseconds);
```

Comment:
```suggestion
  return new Date(seconds * 1000);
```

I think it's simple enough. Or name the variable better because "milliseconds" is not the same as 1000. You could name it `oneSecondInMilliseconds` but I think it's a little overkill